modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -668,6 +696,22 @@ void ParallelCoordinates::axisPicked(PickingEvent* p, size_t pickedID, PickType
         p->markAsUsed();
         invalidate(InvalidationLevel::InvalidOutput);
     }
+    if (p->getPressState() == PickingPressState::DoubleClick &&
+        p->getPressItem() == PickingPressItem::Primary) {
+                
+        axes_[pickedID].pcp->invertRange.set(!axes_[pickedID].pcp->invertRange);
```

Comment:
In my opinion, axis flipping is not so important that it should be assigned to double-click... 
Context-menu option would be great here I think.

Reply:
If a context menu event for a specific mouse position would exist, yes.
However, the double click feature was requested by @Alboin  and @petersteneteg